libcontainer/specconv/spec_linux.go

```diff
@@ -324,18 +324,35 @@ type CreateOpts struct {
 	RootlessCgroups  bool
 }
 
+// getwd is a wrapper similar to os.Getwd, except it always gets
+// the value from the kernel, which guarantees the returned value
+// to be absolute and clean.
+func getwd() (wd string, err error) {
+	for {
+		wd, err = unix.Getwd()
+		//nolint // unix errors are bare
```

Comment:
nit: this should be `//nolint:errorlint`, since we still want all other linters to be enabled.